packages/node/src/index.ts

```diff
@@ -528,20 +531,99 @@ export const startDevServer: StartDevServer = async opts => {
     filename: 'package.json',
   });
   const pkg = pathToPkg ? require_(pathToPkg) : {};
+  const tsNodePath = require_.resolve('ts-node');
+  const esmLoader = join(tsNodePath, '..', '..', 'esm.mjs');
+  const cjsLoader = join(tsNodePath, '..', '..', 'register');
```

Comment:
Interestingly enough this path may need to be more explicit. Based on the `exports` I'm seeing, its resolving `./register/index.js` automatically. We should be explicit like in the `esmLoader` variable. 
```suggestion
  const cjsLoader = join(tsNodePath, '..', '..', 'register', 'index.js');
```